### PR TITLE
Prevent message from being sent to backend if empty, addressing #3

### PR DESCRIPTION
**Pull request by ad-t**

### CODE REVIEW

**Comment by ad-t** [comment]
Hey @kennib, I've added code that prevents the frontend from sending an empty message and prevents the API from digesting an empty message.

File: api.py [hunk 1]
```diff
@@ -247,6 +247,8 @@ def post_message():
 
   db = get_db()
   message = dict(request.values)
+  if message['text'].strip() == "":
```
Comment (kennib) [inline api.py]: Could you change `message['text'].strip()` to `request.values['text'].strip()` and put the changes on line 247, before the database connection is set up?

**Review by kennib — CHANGES_REQUESTED** [review]
Looks good, should be merged once you've made that small change.

File: api.py (hunk 1 above)
Comment (ad-t) [reply api.py]: Done.

**Review by kennib — APPROVED** [review]

**Comment by kennib** [comment]
Nice job @ad-t!

**Pull request merged**